packages/vx-brush/src/BaseBrush.tsx

```diff
@@ -185,36 +177,33 @@ export default class BaseBrush extends React.Component<BaseBrushProps, BaseBrush
   };
 
   handleDragEnd = () => {
-    const { isBrushing } = this.state;
-    if (isBrushing) {
```

Comment:
noting to document: 
by the time this function is called `state` has been updated `isBrushing` usually has been updated to `false` and so this feature didn't work as intended.